app/assets/stylesheets/modules/item-selector.scss

```diff
@@ -13,6 +13,8 @@
     i {
       @extend .glyphicon-tiny;
       @extend .pull-right;
+      float: none !important;
```

Comment:
The float comes from extending the `.pull-right` class.  If you want to remove the float remove the `@extend` line instead.